scripts/windows_dev_setup.ps1

```diff
@@ -409,15 +409,15 @@ function install_dotnet {
     Write-Host "Installing Microsoft DotNet..."
     winget install "Microsoft.DotNet.SDK.$dotnet_version" --accept-source-agreements --silent
 
-    $dotnet_version = winget show -e "Microsoft.DotNet.SDK.$dotnet_version" | Select-String Version
+    $dotnet_version = winget show -e "Microsoft.DotNet.SDK.$dotnet_version" | Select-String -pattern "Version|버전"

```

Comment:
Does winget have an option to output this information in a programmatic way? For example output the data as JSON + where the keys don't change based on the language. As far as I can tell this is just grepping the standard output right now.

Reply:
I tried with different values for `--locale` (https://learn.microsoft.com/en-us/windows/package-manager/winget/show), but there wasn't any luck. It seems that `winget show -e ...` displays keys and values in the standard output, where the keys is in Korean if the Windows system language is Korean.

Reply:
We will want to also handle Chinese and others then in the future... We will need to think about if we can either programmatically do it or convert the input to English to be consistent.